lib/modules/platform/codecommit/index.ts

```diff
@@ -61,52 +56,42 @@ export async function initPlatform({
   password,
   token: awsToken,
 }: PlatformParams): Promise<PlatformResult> {
-  let accessKeyId = username;
-  let secretAccessKey = password;
+  const accessKeyId = username;
+  const secretAccessKey = password;
   let region: string | undefined;
 
-  if (!accessKeyId) {
-    accessKeyId = process.env.AWS_ACCESS_KEY_ID;
+  if (accessKeyId) {
+    process.env.AWS_ACCESS_KEY_ID = accessKeyId;
   }
-  if (!secretAccessKey) {
-    secretAccessKey = process.env.AWS_SECRET_ACCESS_KEY;
+  if (secretAccessKey) {
+    process.env.AWS_SECRET_ACCESS_KEY = secretAccessKey;
+  }
+  if (awsToken) {
+    process.env.AWS_SESSION_TOKEN = awsToken;
   }
 
   if (endpoint) {
     const regionReg = regEx(/.*codecommit\.(?<region>.+)\.amazonaws\.com/);
     const codeCommitMatch = regionReg.exec(endpoint);
     region = codeCommitMatch?.groups?.region;
```

Comment:
Is there any time where `endpoint` is *not* set and region is *not* available?

Reply:
Codecommit works on a region. I believe we should default it to us-east-1.
But ill.check cuz i juz had an exception

Reply:
exception is happening because of the check we added at the start to check connection,
we are calling `listRepositories` once at the INIT in order to check if we have connected to AWS or not.
and it fails without a region. 

NO DEFAULT region